spec/controllers/research_sessions_controller_spec.rb

```diff
@@ -54,8 +54,8 @@
         { research_session_id: 'i-dont-exist', research_session: { name: 'I do not exist' } }
       end
 
-      it '404s' do
-        expect(response.status).to eql(404)
+      it 'redirects to 404 page' do
+        expect(response.status).to eql(302)
```

Comment:
Why a 302 and not a 404?

Reply:
@joelcarr I think we expect the response to equal 302 (URL redirection) because it's doing a redirect to the 404.